packages/cubejs-server-core/src/core/server.ts

```diff
@@ -718,11 +719,11 @@ export class CubejsServerCore {
    * Returns driver instance by a given context
    */
   public async getDriver(
```

Comment:
📓 because it's a public method which can be called only on instantiated class it looks much better to exclude logger from a supertype